stream_create: Move `Add all users` beside `Choose subscribers`.

shubham-padia opened the pull request:
Fixes part of #30690.

<!-- Describe your pull request here.-->


<!-- If the PR makes UI changes, always include one or more still screenshots to demonstrate your changes. If it seems helpful, add a screen capture of the new functionality as well.

Tooling tips: https://zulip.readthedocs.io/en/latest/tutorials/screenshot-and-gif-software.html
-->

**Screenshots and screen captures:**
<img width="701" alt="Screenshot 2024-07-03 at 2 14 41 AM" src="https://github.com/zulip/zulip/assets/13910561/29a37221-5bac-48de-9e11-2cc24c323b72">



<details>
<summary>Self-review checklist</summary>

<!-- Prior to submitting a PR, follow our step-by-step guide to review your own code:
https://zulip.readthedocs.io/en/latest/contributing/code-reviewing.html#how-to-review-code -->

<!-- Once you create the PR, check off all the steps below that you have completed.
If any of these steps are not relevant or you have not completed, leave them unchecked.-->

- [ ] [Self-reviewed](https://zulip.readthedocs.io/en/latest/contributing/code-reviewing.html#how-to-review-code) the changes for clarity and maintainability
      (variable names, code reuse, readability, etc.).

Communicate decisions, questions, and potential concerns.

- [ ] Explains differences from previous plans (e.g., issue description).
- [ ] Highlights technical choices and bugs encountered.
- [ ] Calls out remaining decisions and concerns.
- [ ] Automated tests verify logic where appropriate.

Individual commits are ready for review (see [commit discipline](https://zulip.readthedocs.io/en/latest/contributing/commit-discipline.html)).

- [ ] Each commit is a coherent idea.
- [ ] Commit message(s) explain reasoning and motivation for changes.

Completed manual review and testing of the following:

- [ ] Visual appearance of the changes.
- [ ] Responsiveness and internationalization.
- [ ] Strings and tooltips.
- [ ] End-to-end functionality of buttons, interactions and flows.
- [ ] Corner cases, error conditions, and easily imagined bugs.
</details>

alya commented:
The space around the button text looks weirdly small to me in the screenshot. Can we make it match all the other "Add" buttons if it currently doesn't?

![Screenshot 2024-07-02 at 11 42 02@2x](https://github.com/zulip/zulip/assets/2090066/4e4519c5-2bcb-4ffa-b541-f81e0a961587)

timabbott commented on web/styles/subscriptions.css:
```diff
@@ -266,6 +266,10 @@ h4.user_group_setting_subsection_title {
     padding-left: 5px;
 }
 
+.people-to-add-label {
```
I realized you're riffing off an existing adjacent name, but I think `choose-subscribers-label` would be a lot better name for this.

shubham-padia replied:
Fixed [here](https://github.com/zulip/zulip/compare/d8741f59de03a838e198d293d766333437997381..29af3356d5e69973fa588f17d436b08805086c47)

shubham-padia commented:
> The space around the button text looks weirdly small to me in the screenshot. Can we make it match the other "Add" buttons if it currently doesn't?

That was intentional on my part, I tried to make it smaller in hopes that it would look like a smaller button, I've reverted it to the standard behaviour and updated the screenshot in the original PR description.

timabbott commented:
Updated version looks good, merged after a bit of manual testing. Thanks @shubham-padia!

alya commented:
> That was intentional on my part

Got it. Yeah, I think consistency is more important. Also, please be sure to call out decisions like this one for review in the PR description.
